.github/actions/migrate/action.yml

```diff
@@ -0,0 +1,17 @@
+name: Migrate Nx Workspace
+description: Migrate Nx Workspace to the specified version
+inputs:
+  nx-version:
+    description: Nx version to try to migrate
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Migrate
+      shell: bash
+      run: |
+        echo '{"migrations":[]}' > migrations.json
```

Comment:
As of nx `15.8.0` this is no longer needed. Check [this](https://nx.dev/packages/nx/documents/migrate#ifexists) and comment below.

Reply:
Awesome! 
Applied.